helm/dbp-keycloak/templates/realm-configmap.yaml

```diff
@@ -1,10 +1,12 @@
+{{ if .Values.realm.name }}
```

Comment:
I think the relam.name is the worng value to determinate if this config file is needed